cdap-docs/developer-manual/source/building-blocks/datasets/fileset.rst

```diff
@@ -10,7 +10,7 @@ FileSet Dataset
 
 .. highlight:: java
 
-While real-time programs such as flows normally require datasets with random access, batch-oriented
+While real-time programs such as realtime Spark normally require datasets with random access, batch-oriented
```

Comment:
Add commas. Can we omit "realtime" before Spark? so-
While real-time programs, such as Spark, normally require

Reply:
Added commas for the "such as" subordinate clauses. The sentence now seems like it has too many commas. Should the one before "batch-oriented" be a semicolon now?

Reply:
Oh! We actually don't need the commas I mentioned. My mistake, sorry. Please disregard my previous comment.

Reply:
removed.